### ISSUE TITLE
Tooltip incorrect position after text change.

### ISSUE THREAD

**Issue by SimonOrdo** [twbs/bootstrap#19076]
Tooltip seems to have inconsistent positioned after changing the content via Javascript (on a fixed element).

Fiddle to demonstrate (red element show issue) : https://jsfiddle.net/5dpzaqgo/4/

**Comment by twbs-lmvtfy** [comment]
Hi @SimonOrdo!

You appear to have posted a live example (https://fiddle.jshell.net/5dpzaqgo/4/show/light/), which is always a good first step. However, according to [the HTML5 validator](http://html5.validator.nu), **your example has some validation errors**, which might potentially be causing your issue:
* line 114, column 1 thru column 7: End tag for  ```` body ```` seen, but there were unclosed elements.
* line 104, column 3 thru column 77: Unclosed element ```` div ````.

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by SimonOrdo** [comment]
Example Fixed. 

Tooltip seems to have inconsistent positioned after changing the content via Javascript (on a fixed element).

Fiddle to demonstrate (red element show issue) : https://jsfiddle.net/5dpzaqgo/4/

**Comment by twbs-lmvtfy** [comment]
Hi @SimonOrdo!

You appear to have posted a live example (https://fiddle.jshell.net/5dpzaqgo/4/show/light/), which is always a good first step. However, according to [the HTML5 validator](http://html5.validator.nu), **your example has some validation errors**, which might potentially be causing your issue:
* line 114, column 1 thru column 7: End tag for  ```` body ```` seen, but there were unclosed elements.
* line 104, column 3 thru column 77: Unclosed element ```` div ````.

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by SimonOrdo** [comment]
https://jsfiddle.net/5dpzaqgo/6/

**Comment by twbs-lmvtfy** [comment]
Hi @SimonOrdo!

You appear to have posted a live example (https://fiddle.jshell.net/5dpzaqgo/6/show/light/), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* line 38, column 7: [W013](https://github.com/twbs/bootlint/wiki/W013): Bootstrap version might be outdated. Latest version is at least 3.3.6 ; saw what appears to be usage of Bootstrap 3.3.5

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by SimonOrdo** [comment]
https://jsfiddle.net/5dpzaqgo/7/

**Comment by cvrebert** [comment]
`setContent` and `fixTitle` are not part of the public API. Invoking them directly is not supported. Instead, destroy the old tooltip and create a new one.

**Comment by SimonOrdo** [comment]
Fair enough, but that's not the problem apparently.  

http://stackoverflow.com/questions/35181681/bootstrap-tooltip-alignment-issue-after-update

Bug or as designed?  Are there any workarounds?

**Comment by cvrebert** [comment]
@SimonOrdo Going to need a new example of the problem.

**Comment by SimonOrdo** [comment]
@cvrebert No problem. Here you go.  https://jsfiddle.net/5dpzaqgo/8/

Long tooltip title on the first (red) element causes display problems.

**Comment by cvrebert** [comment]
So, this doesn't seem to be related to changing the text. The result is the same if the new text is set as the initial text:
https://jsfiddle.net/4zswstuj/

**Comment by SimonOrdo** [comment]
Alright, so it's to do with the length of the text then?  You want me to create a new issue?  Is it working as designed?  Is there a workaround?

**Comment by cvrebert** [comment]
WontFix for Bootstrap v3 per https://github.com/twbs/bootstrap/issues/15590#issuecomment-113321235
